docs/src/simulation_tutorial.md

```diff
@@ -0,0 +1,673 @@
+```
+
+### **Theta**
+The meaning of `θ` is a bit less intuitive. In a less complex model (one that only has intercepts for the random effects) or if we supress the correlations in the formula with `zerocorr()` then `θ` describes the relationship between the random effects standard deviation and the standard deviation of the residual term.
+In our `kb07_m` example:
+The residual standard deviation is `680.032`.
+The standard deviation of our first variance component *`item - (Intercept)`* is `364.713`.
+Thus our first `θ` is the relationship: variance component devided by residual standard deviation
+364.713 /  680.032 =  `0.53631`
+
+```@example Main
+kb07_m.θ
+```
+
+We also can calculate the `θ` for variance component *`subj - (Intercept)`*.
+The residual standard deviation is `680.032`.
+The standard deviation of our variance component *`subj - (Intercept)`* is `298.026`.
+Thus, the related θ is the relationship: variance component devided by residual standard deviation
+298.026 /  680.032 =  `0.438252`
+
+```@example Main
+kb07_m.θ
+```
+
+We can not calculate the `θ` for variance component *`item - prec: maintain`* yet, because it includes the correlation of
+*`item - prec: maintain`* and *`item - (Intercept)`*.
+The `θ` vector is the flattened version of the variance-covariance matrix - a lowertrinangular matrix.
+The on-diagonal elements are just the standard deviations (the `σ`'s), If all off-diagonal elements are zero, we can use our
+calculation above. The off-diagonal elements are covariances and correspond to the correlations (the `ρ`'s).
+If they are unequal to zero, as it is in our `kb07`-dataset, we cannot recreate the variance-covariance matrix having the model output.
+We just take it from the model we already fitted
+
+See the two inner values:
+```@example Main
+kb07_m.θ
+```
+
+TODO: NEED HELP HERE, IS THEIR ANY WAY TO DEFINE VARIANCE-COVARIANCE MATRIX YET ?
+=========
+<!---
+vc = VarCorr(kb07_m)
+vc.σρ
+kb07_m.λ
+kb07_m.λ[1]
+kb07_m.λ[2]
+
+
+λitem = LowerTriangular(diagm([1.3, 0.35, 0.75]))
+λsubj = LowerTriangular(diagm([1.5, 0.5, 0.75]))
+
+isapprox(kb07_m.θ,  [flatlowertri(kb07_m.λ[1]); flatlowertri(kb07_m.λ[2])])
+
+[flatlowertri(λitem); flatlowertri(λsubj)]
+
+
+# make a lower triangular matricis
+re_item = create_re(0.5363168233715857,0)
+re_item[4]=0.37133693708531357
+
+re_subj = create_re(0.4382528181348316)
+# make the compact form out of it = is equal to θ 
+vcat( flatlowertri(re_item), flatlowertri(re_subj) )
+--->
+
+## *A simple example*
+
+Having this knowledge about the parameters we can now **simulate data from scratch**
+
+The `simdat_crossed()` function from `MixedModelsSim` lets you set up a data frame with a specified experimental design.
+For now, it only makes fully balanced crossed designs!, but you can generate an unbalanced design by simulating data for the largest cell and deleting extra rows.
+
+TODO: NEED HELP: is that still right? I think it is possible now to make partially crossed designs? 
```

Comment:
Yes, `simdat_crossed` only generates balanced data. You can of course always remove balance by deleting.

One big change is that the old version "expanded" the number of subjects/items so that you would always achieve balance. The new version errors if balance isn't possible with the specified version of subjects/items.